### PR TITLE
Move to new VS SDK build tools

### PR DESCRIPTION
**Pull request by jaredpar**
VSO bug: https://devdiv.visualstudio.com/DevDiv/_workitems?id=378646&_a=edit

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
👍

File: build/Targets/Settings.props [hunk 1]
```diff
@@ -73,7 +73,7 @@
     <IsDev15RC Condition="Exists('$(MSBuildBinPath)\..\..\..\Common7\IDE\Microsoft.VisualStudio.ExtensionEngine.dll')">true</IsDev15RC>
     <IsDev15RC Condition="Exists('$(MSBuildBinPath)\..\..\..\..\Common7\IDE\Microsoft.VisualStudio.ExtensionEngine.dll')">true</IsDev15RC>
     <DeployExtension Condition="'$(IsDev15RC)' != 'true' OR '$(VisualStudioVersion)' != '15.0'">false</DeployExtension>
-    <VisualStudioBuildToolsVersion>15.0.25929-RC2</VisualStudioBuildToolsVersion>
+    <VisualStudioBuildToolsVersion>$(MicrosoftVSSDKBuildToolsVersion)</VisualStudioBuildToolsVersion>
 
     <!-- For the moment our Dev14 VSI suites must continue to use the Dev14 SDK. -->
     <VisualStudioBuildToolsVersion Condition="'$(IsDev14VsiBuild)' == 'true'">14.3.25420</VisualStudioBuildToolsVersion>
```
Comment (Pilchie) [inline build/Targets/Settings.props]: Can we get rid of this (and the use of 14.3.25420 entirely) in the `post-dev15` branch?

Comment (jaredpar) [inline build/Targets/Settings.props]: @tmeschter we are completely done with Dev14 in VSI now correct?

Comment (tmeschter) [inline build/Targets/Settings.props]: @jaredpar Correct.

**Comment by natidea** [comment]
@MattGertz this is targeting post-dev15 so we will not be bringing to shiproom

**Comment by jaredpar** [comment]
@natidea there is a side email discussing this and whether / not to bring to ship room.  If this just worked it's possible we'd bring this.  However we hit several other downstream issues that would block insertion and we're working those out now.

**Comment by MattGertz** [comment]
I would like to fix this if possible, when it's ready.  I don't like being out of sync and it blocks Dogfooding.

**Comment by jaredpar** [comment]
retest this please

**Comment by jaredpar** [comment]
retest this please

**Comment by jaredpar** [comment]
First redo was for myget not being sync'd 
Second redo was for jenkins not being happy with itself

**Comment by jaredpar** [comment]
CC @MattGertz for ship room approval.

**Comment by MattGertz** [comment]
Yep.

**Comment by Pilchie** [comment]
@MattGertz Note that it looks like this issue is also affecting F# https://github.com/Microsoft/visualfsharp/issues/2356

**Comment by natidea** [comment]
Approved to merge via [378646](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/378646)

**Pull request merged**